doc/Command-Reference.md

```diff
@@ -1603,6 +1603,38 @@ Optionally, you can specify an IP address in order to display only that particul
   Click [here](#Quagga-BGP-Show-Commands) to see the example for "show ip bgp neighbors" for Quagga.
 
 
+**show ip bgp network [[<NONE>|<ipv4-address>|<ipv4-prefix>] [(bestpath | multipath | longer-prefixes | json)]]
```

Comment:
I think it is better to have only `[<ipv4-address>|<ipv4-prefix>]`
What do you think? The same for ipv6

Reply:
Agreed. There is no need for <NONE>, as it is all in square brackets, meaning it is all optional.

Reply:
Fixed.